libcloud/common/ovh.py

```diff
@@ -226,7 +226,7 @@ def handle_and_rethrow_user_friendly_invalid_region_error(host, e):
     """
     msg = str(e).lower()
 
-    if 'name or service not known' in msg or 'getaddrinfo failed' in msg:
+    if 'nodename nor servname provided, or not known' in msg or 'getaddrinfo failed' in msg:
```

Comment:
I needed to make a small change to get tests to pass (aka also check for ``'name or service not known' ``). I do agree that checking the error message is not the most robust approach though.

Reply:
hmm interesting, maybe related to a requests version in my local environment or even os-specific (running on mac). changed it